### PR TITLE
chore: updated `@walletconnect` deps

### PR DESCRIPTION
**Pull request by ganchoradkov**
- updated auth examples to latest
- updated sign examples to latest
- updated provider examples to latest
- implemented optional namespace handling to main example wallet

### CODE REVIEW

File: wallets/react-wallet-v2/src/views/SessionProposalModal.tsx [hunk 1]
```diff
@@ -3,36 +3,23 @@ import ProposalSelectSection from '@/components/ProposalSelectSection'
...
+import { Fragment, useEffect, useState } from 'react'
 
 export default function SessionProposalModal() {
   const [selectedAccounts, setSelectedAccounts] = useState<Record<string, string[]>>({})
   const hasSelected = Object.keys(selectedAccounts).length
 
   // Get proposal data and wallet address from store
   const proposal = ModalStore.state.data?.proposal
+  useEffect(() => {
+    console.log('selectedAccounts', selectedAccounts)
```
Comment (bkrem) [inline wallets/react-wallet-v2/src/views/SessionProposalModal.tsx:21]: Wanted or debug only?

**Review by bkrem — APPROVED** [review]
> implemented optional namespace handling to main example wallet

Thank you for this 💯

**Comment by bkrem** [comment]
On a more general note: should we set up renovateBot for web-examples too now that we have it properly configured in monorepo and utils? My only concern is a huge PR wave here whenever we update `@walletconnect/*` pkgs, but I think we can aggregate all into a single PR with https://docs.renovatebot.com/configuration-options/#groupname

File: wallets/react-wallet-v2/src/views/SessionProposalModal.tsx (hunk 1 above)
Comment (ganchoradkov) [reply wallets/react-wallet-v2/src/views/SessionProposalModal.tsx]: Now that the examples have been stable for quite some time, there's no need for the logs

**Comment by ganchoradkov** [comment]
> On a more general note:
> 
> Should we set up renovateBot for web-examples too now that we have it properly configured in monorepo and utils? My only concern is a huge PR wave here whenever we update `@walletconnect/*` pkgs, but I think we can aggregate all into a single PR with https://docs.renovatebot.com/configuration-options/#groupname

We can give it a try and if it works - awesome, if not - will disable it

**Pull request merged**